packages/mdc-shape/_variables.scss

```diff
@@ -20,39 +20,14 @@
 // THE SOFTWARE.
 //
 
-.mdc-shape-container {
-  display: inline-block;
-  position: relative;
-  overflow: hidden; // Prevent overflowed borders from encroaching on surrounding elements
-}
+// Shape categories
```

Comment:
Are these really the correct defaults? Will all 3 of these values end up being used OOTB? It seems like we might end up with more exceptions than inclusions with regard to these "buckets".

Reply:
Yes, these are updated baseline values for shape categories.